soundfile.py

```diff
@@ -216,9 +238,59 @@
     __libsndfile_version__ = __libsndfile_version__[len('libsndfile-'):]
 
 
-def read(file, frames=-1, start=0, stop=None, dtype='float64', always_2d=False,
-         fill_value=None, out=None, samplerate=None, channels=None,
-         format=None, subtype=None, endian=None, closefd=True):
+
```

Comment:
Are all of these overloads just for the various dtypes? That's awfully verbose. I don't like that at all. Can this be compacted?